README.md

```diff
@@ -43,51 +63,51 @@ The Spectrum Web Components documentation site is available via the following co
 yarn docs:start
 ```
 
-By default the resulting site will be available at [http://localhost:8080](http://localhost:8080).
+By default, the resulting site will be available at [http://localhost:8080](http://localhost:8080).
 
-In the case that you'd like to serve and test a static build of the documentation from the root director of a server (`localhost` or otherwise), use:
+In the case that you'd like to serve and test a static build of the documentation from the root directory (`localhost` or otherwise), use:
 
 ```bash
 yarn docs:build
 ```
 
 # Advanced development
 
-There are several more commands that can be useful in specific scenarios:
+There are several commands that can be useful in specific scenarios:
 
 -   `yarn build:clear-cache` to remove previously created artifacts of the `tsc build` process.
 -   `yarn spectrum-vars` to ensure that theme files are up-to-date.
 -   `yarn process-icons` to make sure that the most recent icons are included.
 -   `yarn process-spectrum` to process the spectrum CSS style sources into the individual packages.
--   `yarn build` to make sure the most recent code base is represented in each package's `lib` folders. This happens automatically in the storybook command.
+-   `yarn build` to make sure the most recent code base is represented in each package's `lib` folders. This happens automatically when the storybook command is run.
 
 ## Linting
 
 The project will be linted on a pre-commit hook, but you can also run the lint suite with `yarn lint`. It uses ESLint to lint the JS / TS files, and StyleLint to lint the CSS files.
 
-### Dependency Linting
+### Dependency linting
 
 There are downstream issues that can arise from multiple packages in this mono-repo using dependencies with mismatching version strings. This is particularly an issue for dependencies below `1.0.0` but can be exacerbated in that context and others by more strict settings that can be applied in various package managers. By default, Lerna will bump version numbers of internal dependencies when the various packages are published and the depended version is pointing to the latest release, which can help to mitigate this issue. This can be further mitigated by using `^0.0.0` structured dependency versions, the `^` allowing for the highest amount of upward flexibility in satisfying the dependency. When using these version strings, `yarn lint: versions` which ensure that all instances of those strings for the same dependency match across the repo.
 
 ## Testing
 
 ### Unit Tests
```

Comment:
While we're here, it is "Spectrum" to initial caps the way you are, so is this right English?
```suggestion
### Unit tests
```